tasks/modules.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+
+class GoModule:
+    """A Go module abstraction."""
+
+    def __init__(self, path, targets=None, condition=lambda: True, dependencies=None):
+        self.path = path
+        self.targets = targets if targets else ["."]
+        self.dependencies = dependencies if dependencies else []
+        self.condition = condition
+
+    def __version(self, agent_version):
+        """Return the module version for a given Agent version.
+        >>> mods = [GoModule("."), GoModule("./pkg/util/log")]
+        >>> [mod.__version("7.27.0") for mod in mods]
+        ["v7.27.0", "v0.27.0"]
+        """
+        if self.path == ".":
+            return "v" + agent_version
+
+        return "v0" + agent_version[1:]
```

Comment:
Breaks for Agent 10 :P

Reply:
I think we will catch that when the time comes 😄